gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/pipeline.config

```diff
@@ -138,10 +138,19 @@ train_config {
     }
   }
   data_augmentation_options {
-    random_horizontal_flip {
-    }
+   random_rgb_to_gray {
+    probability : 0.5
   }
-  data_augmentation_options {
+ }
+ data_augmentation_options{
+  random_adjust_saturation{
+  }
+ }
+ data_augmentation_options{
+  random_adjust_brightness{
+  }
+ }
+   data_augmentation_options {
```

Comment:
We don't need to add this to this PR, but I believe it will be useful to add in the future `random_image_scale`, `random_distort_color`, `random_black_patches`, etc, or even use a few from https://github.com/aleju/imgaug

Reply:
(to clarify: this is just suggestions for the future)

Reply:
got it